src/literature/components/LiteratureItem.jsx

```diff
@@ -67,8 +79,7 @@ class LiteratureItem extends Component {
 
 LiteratureItem.propTypes = {
   metadata: PropTypes.instanceOf(Map).isRequired,
-  display: PropTypes.instanceOf(Map).isRequired,
+  display: PropTypes.instanceOf(Map).isRequired
```

Comment:
this is wrong, but probably done by prettier. Can you add `"trailingComma": "es5"`? to .prettierrc.json in this pr